tests/runtime/symlinks/test.wake

```diff
@@ -5,11 +5,13 @@ export def goodLink Unit =
     require Pass dat =
         datFile Unit
     makeExecPlan ("ln", "-s", "datFile", "goodLink", Nil) (dat, Nil)
+    | setPlanLabel "goodLink"
```

Comment:
In lieu of a tool name here, it might be better to label with the role?  Though I definitely don't want to introduce too strong a precedent for that; we definitely do *not* want anything which leads people toward `addLabelContext`.
```suggestion
    | setPlanLabel "test (symlinks): goodLink"
```